src/lib/DriveProvider.jsx

```
@@ -28,7 +28,7 @@ const DriveProvider = ({ client, lang, polyglot, dictRequire, children }) => {
         <DataProxyWrapper isPublic={isPublic}>
           <VaultProvider cozyClient={client}>
             <VaultUnlockProvider>
-              <SharingProvider doctype="io.cozy.files" documentType="Files">
+              <SharingProvider doctype="io.cozy.files" documentType="Files" isPublic={isPublic}>
```

ESLint

[error] Replace `·doctype="io.cozy.files"·documentType="Files"·isPublic={isPublic}` with `⏎················doctype="io.cozy.files"⏎················documentType="Files"⏎················isPublic={isPublic}⏎··············`

(prettier/prettier)